sdk/src/main/scala/com/horizen/api/http/SidechainBlockActor.scala

```diff
@@ -35,6 +35,14 @@ class SidechainBlockActor[PMOD <: PersistentNodeViewModifier, SI <: SidechainSyn
     context.system.eventStream.subscribe(self, classOf[ChangedHistory[HR]])
   }
 
+  override def postStop(): Unit = {
+    log.info("SidechainBlock Actor is stopping...")
+    context.system.eventStream.unsubscribe(self, classOf[SemanticallyFailedModification[PMOD]])
```

Comment:
do we need to unsubscribe to avoid `dead letters` messages?

Reply:
That was I was trying to avoid, but it seems not the case. I think is anyway cleaner unregistering from what we registered in the start phase, for this reason I kept it.

Reply:
I removed all unnecessary unsubscribe() calls